src/view/com/notifications/FeedItem.tsx

```diff
@@ -184,10 +186,24 @@ let FeedItem = ({
     action = _(msg`reposted your post`)
     icon = <RepostIcon size="xl" style={{color: t.palette.positive_600}} />
   } else if (item.type === 'follow') {
+    let isFollowBack = false
     if (
       item.notification.author.viewer?.following &&
-      gate('ungroup_follow_backs')
+      AppBskyGraphFollow.isRecord(item.notification.record)
     ) {
+      try {
+        const rkey = new AtUri(item.notification.author.viewer.following).rkey
+        const followingTimestamp = TID.fromStr(rkey).timestamp()
+        const followedTimestamp =
+          new Date(item.notification.record.createdAt).getTime() * 1000
+        isFollowBack = followedTimestamp > followingTimestamp
+      } catch (e) {
```

Comment:
If this catch exists solely for the AtUri constructor, let's move just the URI construction in it.

```
let rkey
try {
  rkey = ...
} catch (e) {
  // ..
}
if (rkey) {
  // ...
}
```

Otherwise we risk catching random other mistakes like typos or null references that we won't notice.

Reply:
In general, the rule of thumb is we should keep the minimal necessary code that we expect to throw in try/catches like this, and move all `foo.bar.baz` access or user code out of it. Otherwise we're risking swallowing legitimate code mistakes.

Reply:
Yea, this here is fine. Will change.

Reply:
Lets see if prettier complains about that...